docs/ss_slide_004b.md

```diff
@@ -2,10 +2,10 @@
 
 
 {:style="text-align:center;"}
-![lenia title](https://raw.githubusercontent.com/riveSunder/yuca/master/assets/consequential_step_size/adaptive_ss_fail.gif)
+![lenia title](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/consequential_step_size/adaptive_ss_fail.gif)
 The wobbly glider in s613 actually becomes less stable when we try to use an adaptive step size to increase accuracy.
 {:style="text-align:center;"}
 
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/ss_slide_004) -- [Next slide](https://rivesunder.github.io/yuca/ss_slide_005)
+[Previous slide](https://rivesunder.github.io/yuca_docs/ss_slide_004) -- [Next slide](https://rivesunder.github.io/yuca_docs/ss_slide_005)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.